### PR TITLE
Allow excluding wordlists when building for browserify

### PR DESCRIPTION
**Pull request by junderw**
This will help when packaging bip39 to keep file size down.

With all wordlists: 391392 bytes uglified / compressed.

With English only: 249920 bytes

That saves 141472 bytes

### CODE REVIEW

**Comment by junderw** [comment]
This is the TS version of #98

File: ts_src/_wordlists.ts [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+try {
+  _default = require('./wordlists/japanese.json');
+  wordlists.japanese = _default as string[];
+  wordlists.JA = _default as string[];
+} catch (err) {}
+try {
+  _default = require('./wordlists/english.json');
+  wordlists.english = _default as string[];
+  wordlists.EN = _default as string[];
+} catch (err) {}
```
Comment (d-yokoi) [inline ts_src/_wordlists.ts]: I think it would be even better if you implement this using loop.

File: ts_src/index.ts [hunk 2]
```diff
@@ -181,16 +182,9 @@ export function validateMnemonic(
...
-  english: ENGLISH_WORDLIST,
-  french: FRENCH_WORDLIST,
-  italian: ITALIAN_WORDLIST,
-  japanese: JAPANESE_WORDLIST,
-  korean: KOREAN_WORDLIST,
-  spanish: SPANISH_WORDLIST,
-};
+export function setDefaultWordlist(language: string): void {
+  const result = wordlists[language];
+  if (result) DEFAULT_WORDLIST = result;
```
Comment (d-yokoi) [inline ts_src/index.ts]: I expect this function to throw error if the wordlist does not exist.

Comment (d-yokoi) [reply ts_src/index.ts]: In addition, why don't we provide `getDefaultWordlist` as well.

File: ts_src/index.ts [hunk 3]
```diff
@@ -185,6 +185,18 @@ export function validateMnemonic(
 export function setDefaultWordlist(language: string): void {
   const result = wordlists[language];
   if (result) DEFAULT_WORDLIST = result;
+  else
+    throw new Error('Could not find wordlist for language "' + language + '"');
+}
+
+export function getDefaultWordlist(): string {
+  if (!DEFAULT_WORDLIST) throw new Error('No Default Wordlist set');
```
Comment (junderw) [inline ts_src/index.ts]: Maybe this should just return null...?

Comment (d-yokoi) [reply ts_src/index.ts]: I think `undefined` is fine.

Comment (d-yokoi) [reply ts_src/index.ts]: Why don't you export `DEFAULT_LANGUAGE` from `_wordlists.ts` and use it?

Comment (junderw) [reply ts_src/index.ts]: what do you mean?

**Pull request merged**